gcomm/src/gmcast.cpp

```diff
@@ -12,38 +12,38 @@
 #include "defaults.hpp"
 #include "gu_convert.hpp"
 #include "gu_resolver.hpp"
-#include "gu_asio.hpp" // gu::conf::use_ssl
+#include "gu_asio.hpp"  // gu::conf::use_ssl
 
 using namespace std::rel_ops;
 
-using gcomm::gmcast::Proto;
-using gcomm::gmcast::ProtoMap;
 using gcomm::gmcast::Link;
 using gcomm::gmcast::LinkMap;
 using gcomm::gmcast::Message;
+using gcomm::gmcast::Proto;
+using gcomm::gmcast::ProtoMap;
 
 const long gcomm::GMCast::max_retry_cnt_(std::numeric_limits<int>::max());
 
-static void set_tcp_defaults (gu::URI* uri)
```

Comment:
It looks like this file has been reformatted to MySQL style (rather than keeping to galera).

Reply:
Fixed.